src/commands/remoteconfig-get.ts

```diff
@@ -0,0 +1,72 @@
+import * as rcGet from "../remoteconfig/get";
+import { Command } from "../command";
+import { requireAuth } from "../requireAuth";
+import * as logger from "../logger";
+import { RemoteConfigTemplate } from "../remoteconfig/interfaces";
+import getProjectId = require("../getProjectId");
+import { requirePermissions } from "../requirePermissions";
+import { parseTemplateForTable } from "../remoteconfig/get";
+
+import Table = require("cli-table");
+import * as fs from "fs";
+import util = require("util");
+
+const tableHead = ["Entry Name", "Value"];
+
+// Creates a maximum limit of 50 names for each entry
+const MAX_DISPLAY_ITEMS = 20;
+
+function checkValidNumber(versionNumber: string): string {
+  if (typeof Number(versionNumber) == "number") {
+    return versionNumber;
+  }
+  return "null";
+}
+
+module.exports = new Command("remoteconfig:get")
+  .description("Get a Firebase project's Remote Config template")
```

Comment:
don't capitalize the descriptions, please: go/firebase-cli-contributor#create-a-file-for-your-command

```suggestion
  .description("get a Firebase project's Remote Config template")
```